Auto-publishing fails (due to `scip` breaking change?)

```
error[E0063]: missing field `position_encoding` in initializer of `scip::types::Document`
   --> src/cli/scip.rs:184:28
    |
184 |             documents.push(scip_types::Document {
    |                            ^^^^^^^^^^^^^^^^^^^^ missing `position_encoding`

For more information about this error, try `rustc --explain E0063`.
```

https://github.com/rust-lang/rust-analyzer/actions/runs/7692062654/job/20958357191

--- a/crates/rust-analyzer/Cargo.toml
+++ b/crates/rust-analyzer/Cargo.toml
@@ -23,7 +23,7 @@ anyhow.workspace = true
 crossbeam-channel = "0.5.5"
 dissimilar.workspace = true
 itertools.workspace = true
-scip = "0.3.1"
+scip = "0.3.3"
 lsp-types = { version = "=0.95.0", features = ["proposed"] }
 parking_lot = "0.12.1"
 xflags = "0.3.0"

--- a/crates/rust-analyzer/src/cli/scip.rs
+++ b/crates/rust-analyzer/src/cli/scip.rs
@@ -143,11 +143,14 @@ impl flags::Scip {
                             .map(|hover| hover.markup.as_str())
                             .filter(|it| !it.is_empty())
                             .map(|it| vec![it.to_owned()]);
+                        let position_encoding =
+                            scip_types::PositionEncoding::UTF8CodeUnitOffsetFromLineStart.into();
                         let signature_documentation =
                             token.signature.clone().map(|text| scip_types::Document {
                                 relative_path: relative_path.clone(),
                                 language: "rust".to_string(),
                                 text,
+                                position_encoding,
                                 ..Default::default()
                             });
                         let symbol_info = scip_types::SymbolInformation {
@@ -181,13 +184,16 @@ impl flags::Scip {
                 continue;
             }
 
+            let position_encoding =
+                scip_types::PositionEncoding::UTF8CodeUnitOffsetFromLineStart.into();
             documents.push(scip_types::Document {
                 relative_path,
                 language: "rust".to_string(),
                 occurrences,
                 symbols,
-                special_fields: Default::default(),
                 text: String::new(),
+                position_encoding,
+                special_fields: Default::default(),
             });
         }
 

